Italian/finestra/chiudi/chiudi.md

```diff
@@ -0,0 +1,7 @@
+
+Chiudi la finestra e ti ricordi
+
+é ora di andare a dormire
+
+[Torni a dormire](../caramelle.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.